### ISSUE TITLE
rum.core/derived-atom without docs

### ISSUE THREAD

**Issue by madstap** [tonsky/rum#84]
`rum.derived-atom/derived-atom` has a nice docstring, but it doesn't transfer over to `rum.core/derived-atom`, which appears to be undocumented. A fix could be to move the docstring to the `def` in `rum.core`.

**Comment by tonsky** [comment]
Please provide a PR :)

**Comment by madstap** [comment]
When I'm already messing around with this particular fns metadata, would you accept that I add indentation metadata as well, so that emacs (and eventually, hopefully, other editors) indent it like:

``` clojure
  (def *x (derived-atom [*a *b] ::key
            (fn [a b]
              (str a \":\" b))))
```

instead of the current

``` clojure
(def *x (derived-atom [*a *b] ::key
                      (fn [a b]
                        (str a \":\" b))))
```

**Comment by tonsky** [comment]
sure

**Comment by madstap** [comment]
PR provided :)

**Comment by tonsky** [comment]
Thanks! Closed in #85
